Docker/dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3
```

Comment:
Filename should be `Dockerfile` by convention. Folder names should never be uppercase on the other hand, so better `docker`.

Reply:
Changed